ddtrace/span.py

```diff
@@ -474,7 +474,7 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
         # readable version of type (e.g. exceptions.ZeroDivisionError)
         exc_type_str = "%s.%s" % (exc_type.__module__, exc_type.__name__)
 
-        self.meta[ERROR_MSG] = str(exc_val)
+        self.meta[ERROR_MSG] = stringify(exc_val)
```

Comment:
```suggestion
        if isinstance(exc_val, bytes):
            self.meta[ERROR_MSG] = exc_val.decode()
        else:
            self.meta[ERROR_MSG] = stringify(exc_val)
```


I know we have a similar pattern elsewhere in our codebase, but this is highly dependent on the exception value being a `bytes`.